bingosync-app/static/bingosync/style.css

```diff
@@ -84,6 +92,10 @@
     overflow-y: scroll;
 }
 
+.panel-body {
+    padding:8px;
```

Comment:
Yeah, it was mainly because of the lines breaking, I set it to 8px on all of the 'panel-body's because it seemed overall nicer with less padding to me.

If you like the padding the way it is then maybe decreasing the text size for those two panels at the bottom is a better option